src/app/ApplicationTemplate.Mobile/Android/MainActivity.Android.cs

```diff
@@ -1,18 +1,21 @@
-using Android.App;
+﻿using Android.App;
 using Android.Content.PM;
-using Android.OS;
 using Android.Views;
-using Android.Widget;
 
-namespace ApplicationTemplate
+namespace ApplicationTemplate;
+
+[Activity(
+	// We need to set Name, otherwise a random one is generated at every build, which causes the app to be removed from the home screen on app-updates.
```

Comment:
Is this comment still relevant today ?

Reply:
Good catch, I'm not sure, I guess I'll do some research.

Reply:
Not an issue anymore, removed it.